src/components/organisms/EntityList/EntityCreationModal/TestCreationForm/TestCreationForm.tsx

```diff
@@ -123,16 +131,18 @@ const TestCreationForm: React.FC<TestCreationFormProps> = props => {
           shouldUpdate={(prevValues, currentValues) => prevValues.testSource !== currentValues.testSource}
         >
           {({getFieldValue}) => {
-            const testSourceValue = getSourceFieldValue(getFieldValue);
+            const testSourceValue = getSourceFieldValue(getFieldValue) as SourceType;
 
-            const executorType = selectedExecutor?.executor.meta?.iconURI;
+            const executorType = selectedExecutor?.executor.meta?.iconURI || 'unknown';
```

Comment:
why do we need that 'unkown' ? use null instead?

Reply:
<img width="221" alt="image" src="https://user-images.githubusercontent.com/25991946/236811057-31f25b32-e853-4967-8518-51a1908eeeb2.png">